integration-tests/zendeskWebhookAPI.spec.ts

```diff
@@ -22,12 +22,14 @@ const defaultWebhookRequestData: ZendeskWebhookRequest = {
   dateFrom: generateZendeskRequestDate(-60),
   dateTo: generateZendeskRequestDate(-60),
   identifierType: 'event_id',
-  eventIds: '637783 3256',
+  eventIds:
+    'c9e2bf44-b95e-4f9a-81c4-cf02d42c1555 c9e2bf44-b95e-4f9a-81c4-cf02d42cabcd',
   piiTypes: 'drivers_license',
   sessionIds: '',
   journeyIds: '',
   userIds: '',
-  dataPaths: ''
+  dataPaths:
```

Comment:
dataPaths aren't supported yet so I don't think we need to add this?

Reply:
Although if it won't cause any failures I guess we can just leave it there until they are supported.

Reply:
Yes. I added that for consistency, as I updated the others to support the current requirement of the athena lambda.